### PR TITLE
smartcollect readme correction

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference : https://razorpay.com/docs/api/smart-collect

### CODE REVIEW

File: documents/virtualaccount.md [hunk 1]
```diff
@@ -224,6 +250,8 @@ $api->virtualAccount->all($options);
 
 ### Fetch payments for a virtual account
 ```php
+$options = array("count" => 2);
```
Comment (sofiya02) [inline documents/virtualaccount.md:253]: please remove

**Review by sofiya02 — CHANGES_REQUESTED** [review]
Fetch Payment Details using ID and Transfer Method - add parameters

File: documents/virtualaccount.md (hunk 1 above)
Comment (ankitdas13) [reply documents/virtualaccount.md]: removed

Comment (ankitdas13) [reply documents/virtualaccount.md]: removed

**Review by sofiya02 — CHANGES_REQUESTED** [review]
check response of all api as some of the parameter in response is missing and update them

File: documents/virtualaccount.md [hunk 2]
```diff
@@ -286,7 +312,7 @@ $api->payment->fetch($paymentId)->bankTransfer();
 
 | Name  | Type      | Description                                      |
 |-------|-----------|--------------------------------------------------|
-| virtualId*  | string    | The id of the virtual to be updated  |
+| paymentId*  | string    | The id of the payment to be updated  |
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/virtualaccount.md:315]: param updated

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**